code/modules/spells/spell_types/soultap.dm

```diff
@@ -0,0 +1,32 @@
+
```

Comment:
```suggestion
#define HEALTH_LOST_PER_SOUL_TAP 20
```

Reply:
Only if you're going define route instead of variable

Reply:
great idea